rest/src/main/kotlin/route/Route.kt

```diff
@@ -810,7 +810,8 @@ public sealed class Route<T>(
         Route<DiscordGuildApplicationCommandPermissions>(
             HttpMethod.Put,
             "/applications/$ApplicationId/guilds/$GuildId/commands/$CommandId/permissions",
-            DiscordGuildApplicationCommandPermissions.serializer()
+            DiscordGuildApplicationCommandPermissions.serializer(),
+            false
```

Comment:
either remove/deprecate the permission edit routes or change this back, setting `requiresAuthorizationHeader` to `false` is simply wrong (it still requires a authorization header, but not with a bot token)